spec/rest.spec.js

```diff
@@ -98,6 +98,57 @@ describe('rest create', () => {
     expect(objectId).toBeDefined();
   });
 
+  it('allows createdAt and updatedAt to be set with maintenance key', async () => {
+    let obj = {
```

Comment:
Is there a reason why you wrote the test case low-level on the REST controller, instead of just using the public APIs? Testing with the public APIs may make more sense, because that ensures you are testing end-to-end, not just an internal fraction of the process.

Reply:
I was browsing the spec and found other cases that were testing this way, e.g. `rest update` -> `ignores createdAt`. But I'm ok to switch them to use REST API to achieve E2E.

Reply:
Yes, it would be great if you could do that, and just remove the REST tests.

Reply:
If we test E2E functionality with the Parse JS SDK, the tests will not be possible because the SDK transparently ignores `createdAt` and `updatedAt` during object save. See here:

https://github.com/parse-community/Parse-SDK-JS/blob/549b7b3e6e25732ff44a4dcda4f070134be818f6/src/ParseObject.js#L710-L714C7

I will proceed with constructing HTTP requests to achieve E2E (from server's perspective) coverage.

Reply:
That sounds good